bulletroboy/operator.py

```diff
@@ -22,8 +22,14 @@ def __init__(self, body_id):
 		self.links = self.get_links()
 		self.movements = Movements(self)
 		self.ef_publisher = self.create_publisher(PoseStamped, '/roboy/simulation/operator/pose/endeffector', 10)
+		self.link_id_service = self.create_service(LinkIdFromName, '/roboy/simulation/operator/link_id_from_link_name', self.get_link_id_from_link_name)
```

Comment:
Variable name of service not used. Im not sure if its better to leave it there or not. Just in case I will mark all the places where this happens. You can decide later whether to change them or not.